generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -84,6 +84,14 @@ def _handle_rpc_error(self, rpc_error):
         else:
             self._raise_error(error_code, error_message, samps_per_chan_written, samps_per_chan_read)
 
+    def _check_for_error_from_response(self, error_code, samps_per_chan_read):
+        if error_code != 0:
```

Comment:
How much overhead does calling GetErrorString() add? (Check NI IO Trace.)

Timeout errors are normal program flow, not exceptional cases, so it may make sense to have a special case that returns a hardcoded copy of the timeout error string. (The DAQmx C API does not currently support localization.)

Reply:
FYI, here are my timing numbers from when we discussed this over Teams chat:

```
>>> import nidaqmx._library_interpreter
>>> interpreter = nidaqmx._library_interpreter.LibraryInterpreter()
>>> interpreter.get_error_string(-200284)
'Some or all of the samples requested have not yet been acquired.\nTo wait for the samples to become available use a longer read timeout or read later in your program. To make the samples available sooner, increase the sample rate. If your task uses a start trigger,  make sure that your start trigger is configured correctly. It is also possible that you configured the task for external timing, and no clock was supplied. If this is the case, supply an external clock.'
>>> import timeit
>>> timeit.timeit(lambda: interpreter.get_error_string(-200284), number=1000) / 1000
4.499499999980117e-06
```
```
>>> import nidaqmx._grpc_interpreter
>>> import grpc
>>> grpc_channel = grpc.insecure_channel("localhost:31763")
>>> grpc_options = nidaqmx.GrpcSessionOptions(grpc_channel, "")
>>> grpc_interpreter = nidaqmx._grpc_interpreter.GrpcStubInterpreter(grpc_options)
>>> timeit.timeit(lambda: grpc_interpreter.get_error_string(-200284), number=1000) / 1000
0.0007822051999999644
```